cfy_manager/components/validations.py

```diff
@@ -513,6 +513,39 @@ def validate_config_access(write_required):
                     USER_CONFIG_PATH, label))
 
 
+def _validate_not_reusing_removed_passwords():
+    # Passwords that we remove after use should not accept the 'removed' value
```

Comment:
comments like this should just be written as docstrings

Reply:
Docstringed.